src/greyVest/TagsInput.js

```diff
@@ -21,19 +21,33 @@ let TagsInput = forwardRef(
       onBlur = _.noop,
       onInputChange = _.noop,
       onTagClick = _.noop,
+      maxWordsPerTag = 100,
+      maxCharsPerTagWord = 100,
       Tag = DefaultTag,
       ...props
     },
     inputRef
   ) => {
     let containerRef = React.useRef()
     let state = useLocalStore(() => ({ currentInput: '' }))
+
+    // Convert string to words, take the first maxWordsPerTag, truncate them and convert back to string
+    let sanitizeWords = _.flow(
+      _.words,
+      _.take(maxWordsPerTag),
+      _.map(word =>
+        _.truncate({ length: maxCharsPerTagWord, omission: '' }, word)
+      ),
```

Comment:
nit: you can omit `word` entirely because `_.truncate` is curried

Reply:
done